dipy/denoise/localPCA_denoise.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from dipy.denoise.rician_adaptation import rician_adaptation
+
+def localPCA_denoise(arr, sigma, patch_radius=1, tou=0, rician=True):
```

Comment:
Please change the name of the file to localpca.py
And the name of the function to localpca. We do not use capital letters in function names.